com.github.git-cola.yml

```diff
@@ -0,0 +1,51 @@
+app-id: com.github.git-cola
+runtime: org.kde.Platform
+runtime-version: '5.15'
+sdk: org.kde.Sdk
+base: io.qt.qtwebkit.BaseApp
+base-version: "5.15"
+command: git-cola
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --filesystem=home
+
+modules:
+  - name: pyside2
+    buildsystem: simple
```

Comment:
you should be able to use `buildsystem: cmake-ninja` and have the first line in a `shell` source. that was always run at unpack time.